server/main_server/src/rodsServer.cpp

```diff
@@ -718,13 +718,26 @@ namespace
                 }
             }
 
-            char hostname[HOST_NAME_MAX]{};
+            char hostname[HOST_NAME_MAX + 1]{};
```

Comment:
this should be part of the next commit?

Reply:
Well, this commit becomes broken if moved into the next commit.

This change sparked the next commit.

Reply:
fair enough, so leave as is... or reshuffle the commits if that helps.   all good.

Reply:
After looking at the commit again, this line can be moved into the other commit.

Swizzling things around.

Reply:
Done.